WordPress/Classes/ViewRelated/Pages/PageListViewController.swift

```diff
@@ -760,6 +766,21 @@ class PageListViewController: AbstractPostListViewController, UIViewControllerRe
         super.deletePost(apost)
     }
 
+    private func addBlazeAction(to controller: UIAlertController, for page: AbstractPost) {
+        guard FeatureFlag.blaze.enabled && page.canBlaze else {
+            return
+        }
+
+        let buttonTitle = NSLocalizedString("pages.blaze.actionTitle", value: "Promote with Blaze", comment: "Promote the page with Blaze.")
+        controller.addActionWithTitle(buttonTitle, style: .default, handler: { [weak self] _ in
+            if let page = self?.pageForObjectID(page.objectID) {
+                self?.blazePage(page)
+            }
```

Comment:
Since `func blazePage(page: Page)` only needs `Page.postID`, I think we can just pass the AbstractPost instead of performing a core data fetch. What do you think?

Reply:
Fixed! [f71efee](https://github.com/wordpress-mobile/WordPress-iOS/pull/20212/commits/f71efee157f7a3cd8d3d4491dc581087aa695a51)